pyrightconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "include": [
+    "src",
+    "tests"
+  ],
+  "typeCheckingMode": "standard",
+  "exclude": [
```

Comment:
It would be good to exclude almost all adapters from here. And selective include the default adapter. Any way to achieve that?